Manual.md

```diff
@@ -1546,6 +1546,12 @@ dependencies at build time. If you notice warnings about `EasyInstall` deprecati
 present in `${wrksrc}/.eggs` after building the package, then those packages should be added to
 `hostmakedepends`.
 
+Python packages that use [Poetry](https://github.com/python-poetry/poetry-core)
+as a build backend should specify the `python3-pep517` build style, add
+`python3-poetry-core` to `hostmakedepends` and approximately map
+`tool.poetry.dependencies` entries in `pyproject.toml` to `depends` in the
+template.
+
```

Comment:
I don't think the manual should call out poetry-core here. It is not the only PEP-517 builder and doesn't deserve special mention, especially because nothing in the repository currently uses it. To be somewhat generic, I'd recommend dropping this paragraph and adding a sentence at the end of the first paragraph (Line 1534) along the lines of

    The `python3-pep517` build style does not depend on a specific build backend, so packages will need to add an appropriate backend provider to `hostmakedepends`.

I don't think the dependency mapping content is important here; discussion of the `setuptools` mapping was included because `setuptools` has a nasty tendency of fetching and locally installing everything needed to build a package. For the `pep517` build style, we explicitly disable fetching to force hard errors when dependencies are improperly specified.

Reply:
Makes sense--fixed in 6d524b.